src/diffusers/models/unets/unet_motion_model.py

```diff
@@ -35,24 +39,1118 @@
 )
 from ..embeddings import TimestepEmbedding, Timesteps
 from ..modeling_utils import ModelMixin
-from ..transformers.transformer_temporal import TransformerTemporalModel
+from ..resnet import Downsample2D, ResnetBlock2D, Upsample2D
+from ..transformers.dual_transformer_2d import DualTransformer2DModel
+from ..transformers.transformer_2d import Transformer2DModel
 from .unet_2d_blocks import UNetMidBlock2DCrossAttn
 from .unet_2d_condition import UNet2DConditionModel
-from .unet_3d_blocks import (
-    CrossAttnDownBlockMotion,
-    CrossAttnUpBlockMotion,
-    DownBlockMotion,
-    UNetMidBlockCrossAttnMotion,
-    UpBlockMotion,
-    get_down_block,
-    get_up_block,
-)
-from .unet_3d_condition import UNet3DConditionOutput
 
 
 logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
 
 
+@dataclass
+class AnimateDiffTransformer3DOutput(BaseOutput):
```

Comment:
Since transfomer is now a block, we don't need to define an output class for it.